examples/glue/glue_entrypoint.ipynb

```diff
@@ -0,0 +1,370 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "❗ Note: The entrypoint runs all 8 GLUE fine-tuning tasks on every saved pre-training checkpoint, so set your [save_interval](https://docs.mosaicml.com/en/stable/trainer/checkpointing.html?highlight=sve%20interval#save-interval) within your `pretrain_hparams` appropriately to avoid unnecessarily long evaluation times. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Executing your job\n",
+    "\n",
+    "Let's now put together all our knowledge about the entrypoint and launch a job that will fine-tune a pre-trained BERT model on the 8 GLUE tasks! Because we are only fine-tuning with no special configurations, we only need to specify our bucket information and the `finetune_ckpts` to load from:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = {\n",
+    "  'finetune_hparams': {\n",
+    "    'loggers':\n",
+    "      {'object_store': {'object_store_hparams': {'s3': {'bucket': 'mosaicml-internal-checkpoints-bert'}}}},\n",
+    "    'load_object_store': {'s3': {'bucket': 'mosaicml-internal-checkpoints-bert'}},\n",
+    "    'save_folder': 'checkpoints',\n",
+    "    'finetune_ckpts': ['bert-baseline-tokenizer-2uoe/checkpoints/ep0-ba68796-rank0']\n",
+    "  }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's now dump our constructed hparams to a YAML file to be loaded by the entrypoint:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/tmp/tmpi1pw7cg_\n"
+     ]
+    }
+   ],
+   "source": [
+    "import yaml\n",
+    "import tempfile\n",
+    "\n",
+    "tmp_file = tempfile.NamedTemporaryFile()\n",
+    "with open(tmp_file.name, 'w+') as f:\n",
+    "    yaml.dump(data, f)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's launch it! At the end of training, we will see a table containing the GLUE per-task, GLUE-Large, and GLUE-All scores!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 47,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.chdir('../../') # change to root composer directory to access full composer tree (ONLY RUN ONCE!)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 49,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "python: can't open file '//examples/glue/run_glue_trainer.py': [Errno 2] No such file or directory\n"
```

Comment:
When running this in Colab I get an error:
`can't open file 'examples/glue/run_glue_trainer.py': [Errno 2] No such file or directory`
I suspect this is because the notebook only runs pip install, but does not pull the core itself?

Reply:
Good catch, added git cloning!